packages/test/src/config.js

```diff
@@ -43,11 +46,13 @@ module.exports = {
     '<rootDir>/node_modules',
     `${cwd}/node_modules/@theforeman/vendor-core/node_modules`,
     `${cwd}/node_modules`,
+    `${cwd}/node_modules/react-virtualized/node_modules`,
+    `${cwd}/node_modules/htmlparser2/node_modules`,
   ],
   setupFiles: [
     'raf/polyfill',
     'jest-prop-type-error',
-    './test/test_setup.js',
+    './src/test_setup.js',
```

Comment:
No need to use the `<rootDir>` here?
I would prefer to use it anyway so it's clear when it's `cwd` and when it is `root`